src/pages/index.js

```diff
@@ -1,84 +1,188 @@
 import Card from "../components/scripts/Card.js";
 import FormValidation from "../components/scripts/FormValidator.js";
-// import Popup from "../scripts/Popup.js";
 import PopupWithForm from "../components/scripts/PopupWithForm.js";
 import PopupWithImage from "../components/scripts/PopupWithImage.js";
 import UserInfo from "../components/scripts/UserInfo.js";
 import Section from "../components/scripts/Section.js";
 import * as DOM from "../utils/constants.js";
 import { validationSettings } from "../utils/utils.js";
-import { initialCards } from "../utils/utils.js";
-
+import Api from "../components/scripts/Api.js";
+import ConfirmationOnClick from "../components/scripts/popupWithConfirmation";
 import "../pages/index.css";
 
+// Api
+const api = new Api({
+  baseUrl: "https://around-api.en.tripleten-services.com/v1",
+  headers: {
+    authorization: "d081ca18-dd96-4ff1-863a-c195fbcda74a",
+    "Content-Type": "application/json",
+  },
+});
+
+const userInformation = new UserInfo(
+  ".profile__title",
+  ".profile__description",
+  ".profile__pic"
+);
+
+const formValidators = {};
+const enableValidation = (validationSettings) => {
+  DOM.forms.forEach((formElement) => {
+    const validator = new FormValidation(validationSettings, formElement);
+    const formName = formElement.getAttribute("name");
+    formValidators[formName] = validator;
+    validator.enableValidation();
+  });
+};
+
+let section = 0;
```

Comment:
`= 0;`

The initial value should be correct (it's not about numbers)


If it should be defined on start 

```js

let section;

```